From e39bc52671fb33e7d79e58cd5838be75e12abeb8 Mon Sep 17 00:00:00 2001
From: Antonio Quartulli <ordex@autistici.org>
Date: Sun, 18 Aug 2013 12:40:03 +0200
Subject: [PATCH 3/4] batman-adv: fix variable name in compat code

compat code introduced by
0d941e82ab5f92faf33bee6abdde519056f3ac2d
("batman-adv: Unmap fragment page once iterator is done")
is declaring a variable having the same name of one used in
the batman-adv code. Rename it to prevent sparse warnings.

Signed-off-by: Antonio Quartulli <ordex@autistici.org>
Signed-off-by: Marek Lindner <lindner_marek@yahoo.de>
---
 compat.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/compat.h b/compat.h
index 346a824..35ed4d9 100644
--- a/compat.h
+++ b/compat.h
@@ -309,10 +309,10 @@ static int __batadv_interface_set_mac_addr(x, y)
 /* older kernels still need to call skb_abort_seq_read() */
 #define skb_seq_read(consumed, data, st) \
 	({ \
-		int len = skb_seq_read(consumed, data, st); \
-		if (len == 0) \
+		int __len = skb_seq_read(consumed, data, st); \
+		if (__len == 0) \
 			skb_abort_seq_read(st); \
-		len; \
+		__len; \
 	})
 #endif /* < KERNEL_VERSION(3, 11, 0) */
 
-- 
1.8.4.rc3

